server/server.js

```diff
@@ -66,35 +66,36 @@ function renderDefaultPage (req, res, html) {
 }
 
 if (config.ResourcePrefix !== 'local') {
-  console.log('\nthing', config.ResourcePrefix, '\n')
   app.get('/', async function (req, res) {
     info(`serving resources from ${config.ResourcePrefix}`)
     req.metricsPath = '/catchall'
     const resourcePath = `${config.ResourcePrefix}${config.ResourceVersion}${req.path}`
-    console.log('resourcePath', resourcePath, '\n\n')
     await _wget(resourcePath).then(html => { renderDefaultPage(req, res, html) })
   })
   app.get('*', async function (req, res) {
     info(`serving resources from ${config.ResourcePrefix}`)
     req.metricsPath = '/catchall'
     const resourcePath = `${config.ResourcePrefix}${config.ResourceVersion}${req.path}`
-    if (!req.path.includes('_next/webpack-hmr')) {
+    if (!req.path.includes('-hmr')) {
       await stream(resourcePath, null, res)
     } else {
       res.sendStatus(404)
     }
   })
 } else {
-  info('using local resources from "../build"')
+  info('using local resources from "../ui/dist"')
   app.get('/', async (req, res) => {
-    const filePath = join(__dirname, '../', 'build', 'index.html')
+    const filePath = join(__dirname, '../ui', 'dist', 'index.html')
+    console.log('filePath', filePath)
     const html = await readFile(filePath)
     renderDefaultPage(req, res, html)
   })
-  app.get('*', _static(join(__dirname, '../build')))
+  app.get('*', _static(join(__dirname, '../ui/dist')))
 }
 
+// app.listen(config.Port, config.Env === 'prod' ? '0.0.0.0' : 'localhost', () => {
 app.listen(config.Port, () => {
-  const message = `Server is running on port ${config.Port}, env: ${config.Env}`
+  const message = `Server is running on port ${config.Port}, env: ${config.Env}\n`
+  console.log(message)
```

Comment:
Do we need both an info and a console.log?

Reply:
I added this so it is clearer that the server is running, and on which port, when you run this from the terminal. The original output shows this info, but it is buried in an object, and is easy to miss, especially for someone new to the project. However, if this is bad form, I'll remove it.

Reply:
Sounds like it's worth keeping it